Proctor_Brad/Assignments/bubble sort.py

```diff
@@ -0,0 +1,18 @@
+import random
+import time
+b = []
+for x in range(0,100):
+    b.append(int(random.random()*10000))
+
+maximum = len(b) - 1
+start_time = time.time()
+for i in range(0,maximum):
+    for j in range(0,maximum):
+        if(b[j] > b[j+1]):
+            temp = b[j]
```

Comment:
check out the tuples section to see the Pythonic method of swapping two variables using only a single line of code